### ISSUE TITLE
Should not try to bundle CSS and other asset files

### ISSUE THREAD

**Issue by swernerx** [ai/size-limit#14]
I think size-limit should not try to bundle CSS files and other references asset files which are typically bundled using configured loaders.

Background: Our component library is using CSS modules and is exported as library where the CSS references are kept in-tact. This should probably not throw with tools like "size-limit". 

I see that there is already a small section for image assets going to the file-loader:

https://github.com/ai/size-limit/blob/master/index.js#L35

We could extend this with far more file types e.g. webfonts and/or add a custom css-loader based section for CSS files.

At least size-limit should not throw errors when reaching these super typical files when doing bundling with webpack.

**Comment by swernerx** [comment]
That's the exception thrown right now:

```
> size-limit
 ERROR  Error: 
ERROR in ./lib/Core-DnMX9gqv.css
Module parse failed: /Users/Sebastian/Workspace/sebastian-software/edge-core/lib/Core-DnMX9gqv.css Unexpected token (14:0)
You may need an appropriate loader to handle this file type.
|  */
| 
| ::before,
|  ::after {
|    background-repeat: no-repeat; /* 1 */
 @ ./lib/browser.es5.esm.js 13:0-44
 @ multi ./lib/browser.es5.esm.js
    at runWebpack.then.stats (/Users/Sebastian/Workspace/sebastian-software/edge-core/node_modules/size-limit/index.js:110:15)
    at <anonymous>
```

**Comment by ai** [comment]
Why we should not bundle them? They affects to bundle size and developers should take care.

Totally agree that it should not throw error. Let's add config for `css-loader`. Do you want to make PR?

**Comment by swernerx** [comment]
Both is okay in my opinion. If size-limit is mainly about JS size then it should respect JS only.

I mean right now you basically exclude asset files as well, as file-loader is replacing them with URLs.

I am not sure what is the right approach here.

If you like to have CSS included in the bundle, I'll do that PR.

**Comment by ai** [comment]
Are you sure, that file-loader replaces then to URL? I thought url-loader does it.

**Comment by swernerx** [comment]
It's the other way around.

"The url-loader works like the file-loader, but can return a DataURL if the file is smaller than a byte limit." via https://github.com/webpack-contrib/url-loader

**Comment by swernerx** [comment]
I am not sure about assets being added to the size. Sometimes you e.g. offer alternative file formats e.g. woff and woff2. I figure most people would never inline all of them.

**Comment by swernerx** [comment]
Here we go: https://github.com/ai/size-limit/pull/15

**Comment by ai** [comment]
Released in 0.10
